yarn.lock

```diff
@@ -2524,6 +2525,36 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@substrate/connect-extension-protocol@npm:^1.0.0":
+  version: 1.0.0
+  resolution: "@substrate/connect-extension-protocol@npm:1.0.0"
+  checksum: a6f16f1b986eb3d517a8db909d780febc1f5094a2956c1d3f9332ee60e0c08f32f67f4f500c9522bacd166d63a5023738f90c28059768077bf26535dc5a82013
+  languageName: node
+  linkType: hard
+
+"@substrate/connect@npm:^0.6.2":
+  version: 0.6.2
+  resolution: "@substrate/connect@npm:0.6.2"
+  dependencies:
+    "@polkadot/rpc-provider": ^7.10.1
```

Comment:
This bit seems problematic? It needs to be kept in alignment?

Reply:
yeah! we are in a bit of a chicken and egg situation here... Once `@substrate/connect` stops exporting the polkadot-js rpc provider, then that dependency will stop being there. Although, if you have closer look at this PR, then you will notice that the parts of the API that are currently being used don't rely on `@polkadot/rpc-provider`.